frontend/src/lib/utils.tsx

```diff
@@ -1166,6 +1166,15 @@ export function lightenDarkenColor(hex: string, pct: number): string {
     return `rgb(${[r, g, b].join(',')})`
 }
 
+export function interpolateHsl(from: [number, number, number], to: [number, number, number], pct: number): string {
```

Comment:
I'm guessing this is copied/adapted from some source... should we credit the source (assuming I'm right)?

Reply:
Hmm, there is a source… GitHub Copilot 🤔 I don't think there is consensus on whether that should be credited

Reply:
I wondered if that was it. Doesn't feel helpful to the future traveller to credit that